src/imfilter.jl

```diff
@@ -841,6 +841,11 @@ function _imfilter_fft!(r::AbstractCPU{FFT},
 end
 
 function filtfft(A, krn)
```

Comment:
Sorry for the absence @kunzaatko.

Roughly speaking, JuliaImages treats two families of types as numeric types: `Colorant{<:Number}` and `T<:Number`.
Thus it's a good idea for the dispatch design to explicitly express this.

I'd like to see three methods:

```julia
# simple channelview/colorview wrapper over the plain number methods
filtfft(A::AbstractArray{CT}, krn) where CT<:Colorant 

filtfft(A::AbstractArray{T}, krn) where T<:AbstractComplex # your newly added method
filtfft(A::AbstractArray{T}, krn) where T<:Real
```

Reply:
No worries. Thank you for your work on this package!
I would argue that we are missing the methods for complex kernels in the two bottom methods. I implemented these in the new version.